### DISCUSSION TITLE
What do the graphs shown on the Android app mean?

### DISCUSSION THREAD

**Discussion by chrisroge** [OpenSeizureDetector/Android_Pebble_SD#77] [02_Q&A] [answered]
Some documentation explaining what story these graphs are meant to tell.  Apologies this exists and I simply did not look hard enough.

**Comment by jones139** [comment, marked as answer]
Yes, this project is lacking a few things, including documentation, sorry :).   The best description of the detection algorithm is here:  https://www.openseizuredetector.org.uk/?page_id=455.

In terms of the graphs, the main bar graph is the frequency spectrum in 1Hz bins so you can see which frequency band has the most movement (the system is set to detect movement in the range 3-8 Hz.
The horizontal graphs show how far off alarming it is for the two conditions that have to be satisfied to create an alarm state:
Power (Alarm Threshold setting)= absolute amount of movement - if there is very little movement, we do not  want to generate an alarm, because we are probably just detecting noise.
Spectrum Ratio (Alarm Ratio Threshold setting) = proportion of the power in the 3-8 Hz compared to the rest of the spectrum - we are looking for an excess of movement in that range compared to other frequencies.   The ratio numbers are a bit odd because they are multiplied by 10 (so a ratio threshold of 54 means 5.4), so we need to see 5.4 times as much power in the 3-8Hz range compared to the whole spectrum average.
Seizure Probability is only visible on V4.1.x versions and is the probability of the data representing a seizure, as calculated by the Machine Learning (AI) model.

Thanks for being interested in how it works!

Graham.
